.github/workflows/deploy_to_environment_azure.yml

```diff
@@ -0,0 +1,171 @@
+name: Deploy to environment (CIP)
```

Comment:
All makes sense to me. Thanks you.
As a nice to have enhancement do you know if there is a way to make it really obvious when viewing the summary of the run whether it was run for staging or production.
e.g. from this view

![image](https://user-images.githubusercontent.com/109524605/206146483-ae0cd4bb-04a8-47e1-bcad-a5920c0ea43e.png)

Reply:
Github released "[Action Summaries](https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/)" which seems like a perfect use for this. It supports conditions, contexts and variables and all rendered with GitHub flavoured markdown!